IDEAS/Buildings/Components/Interfaces/PartialSurface.mo

```diff
@@ -250,6 +257,188 @@ equation
           fillPattern=FillPattern.Solid)}));
 end PowerLaw_q50;
 
+
+
+
+
+model PowerLaw_q50_stack
+
+    replaceable package Medium = Modelica.Media.Interfaces.PartialMedium  constrainedby
+      Modelica.Media.Interfaces.PartialMedium                                                  annotation (
+          __Dymola_choicesAllMatching=true);
+
+
+      parameter Modelica.SIunits.Area A
+             "Surface area";
+
+      parameter Real m=0.65;
+      final parameter Boolean StackEff= sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.TwoPorts "True if stack effect is used";
+
+      /*ERROR:  Current version of the Modelica translator can only handle
+*/
+      final parameter Boolean ColApos=StackEff and h_a>0;
```

Comment:
@kldjonge can you try to add `annotation(Evaluate=true)` to the definition of these variables to try to fix your issue?

Reply:
I tried but it doesn't seem to solve the error.